esi_report.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# run ./esi_report.sh daily to get daily report
+# run ./esi_report.sh monthly to get last month's report
+# run ./esi_report.sh weekly to get last week's report
+# run ./esi_reports.sh plan to get future plan
+
+report_file=/tmp/esi_report
+
+source /home/stack/overcloud-deploy/overcloud/overcloudrc
+
+write_output () {
+    openstack esi lease list -f value |
```

Comment:
I'd format these as '-f csv'. That formatting also generates the header row, so you no longer have to manually generate those below.

Reply:
Got it!